src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/LimitedPartnerControllerValidationTest.java

```diff
@@ -27,7 +35,10 @@
 @WebMvcTest(controllers = {LimitedPartnerController.class})
 class LimitedPartnerControllerValidationTest {
 
-    private static final String POST_URL = "/transactions/863851-951242-143528/limited-partnership/limited-partner";
+    private static final String LIMITED_PARTNER_ID = "863851-951242-143528";
+
+    private static final String POST_URL = "/transactions/" + LIMITED_PARTNER_ID + "/limited-partnership/limited-partner";
```

Comment:
Prefer BASE_URL or something similar instead of POST_URL as it's used to specify the validation-status url which is not POST.

Reply:
Done. Change also applied to the `GeneralPartnerControllerValidationTest` where the same names were used.